### PR TITLE
Improve wording of emoji tables

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @KrauseFx</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -41,13 +41,13 @@ Then to deploy a new 'beta' version of your app just run
...
 :email: | Automatically pass on information from one build step to another (*e.g.* path to the `ipa` file)
-:page_with_curl: | Store **everything** in Git. Never lookup build commands in `Jenkins` configs again.
+:page_with_curl: | Store **everything** in Git. Never lookup build commands on your CI system again.
 :rocket: | Saves you **hours** for every app update you release
 :pencil2: | Flexible configuration using a fully customisable `Fastfile`
 :mountain_cableway: | Implement a fully working Continuous Delivery process
 :ghost: | [Jenkins Integration](https://docs.fastlane.tools/best-practices/continuous-integration/#jenkins-integration): Show output directly in test results
 :book: | Automatically generate Markdown documentation of your lane configurations
-:hatching_chick: | Over 170 built-in integrations available
+:hatching_chick: | Over 170 built-in integrations and 170 plugins available
```
Comment (ohayon) [inline README.md]: Could this sentence be `Over 170 built-in integrations and plugins available` ?

File: fastlane/README.md [hunk 2]
```diff
@@ -85,13 +85,13 @@ fastlane release
...
 :email: | Automatically pass on information from one build step to another (e.g. path to the `ipa` file)
-:page_with_curl: | Store **everything** in git. Never again lookup the build commands in the `Jenkins` configs
+:page_with_curl: | Store **everything** in Git. Never lookup build commands on your CI system again.
 :rocket: | Saves you **hours** for every app update you release
 :pencil2: | Very flexible configuration using a fully customisable `Fastfile`
 :mountain_cableway: | Implement a fully working Continuous Delivery process
 :ghost: | [Jenkins Integration](https://docs.fastlane.tools/best-practices/continuous-integration/#jenkins-integration): Show the output directly in the Jenkins test results
 :book: | Automatically generate a markdown documentation of your lane config
-:hatching_chick: | Over 170 built-in integrations available
+:hatching_chick: | Over 170 built-in integrations and 170 plugins available
```
Comment (ohayon) [inline fastlane/README.md]: Same comment as above.

**Review by ohayon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (KrauseFx) [inline README.md]: Sure, what do you think is better? Maybe it should be `Over 340 built-in integrations and plugins available`

**Comment by KrauseFx** [comment]
@ohayon 👍

**Comment by KrauseFx** [comment]
I'll merge for now, feel free to submit a follow-up PR 👍

**Pull request merged**